c/libuv/error.go

```diff
@@ -0,0 +1,111 @@
+package libuv
+
+import (
+	"github.com/goplus/llgo/c"
+	_ "unsafe"
+)
+
+const (
+	E2BIG Errno = iota
```

Comment:
already defined in https://pkg.go.dev/github.com/goplus/llgo/c/syscall. are they the same?

Reply:
I have modified it to this now: 
```Go
const (
	E2BIG           Errno = (Errno)(syscall.E2BIG)
	EACCES          Errno = (Errno)(syscall.EACCES)
	EADDRINUSE      Errno = (Errno)(syscall.EADDRINUSE)
	EADDRNOTAVAIL   Errno = (Errno)(syscall.EADDRNOTAVAIL)
	EAFNOSUPPORT    Errno = (Errno)(syscall.EAFNOSUPPORT)
	EAGAIN          Errno = (Errno)(syscall.EAGAIN)
	EAI_ADDRFAMILY  Errno = net.EAI_ADDRFAMILY
	EAI_AGAIN       Errno = net.EAI_AGAIN
	EAI_BADFLAGS    Errno = net.EAI_BADFLAGS
	EAI_BADHINTS    Errno = net.EAI_BADHINTS
	EAI_CANCELED    Errno = -3003
	EAI_FAIL        Errno = net.EAI_FAIL
	EAI_FAMILY      Errno = net.EAI_FAMILY
	EAI_MEMORY      Errno = net.EAI_MEMORY
	EAI_NODATA      Errno = net.EAI_NODATA
	EAI_NONAME      Errno = net.EAI_NONAME
	EAI_OVERFLOW    Errno = net.EAI_OVERFLOW
	EAI_PROTOCOL    Errno = net.EAI_PROTOCOL
	EAI_SERVICE     Errno = net.EAI_SERVICE
	EAI_SOCKTYPE    Errno = net.EAI_SOCKTYPE
	EALREADY        Errno = (Errno)(syscall.EALREADY)
	EBADF           Errno = (Errno)(syscall.EBADF)
	EBUSY           Errno = (Errno)(syscall.EBUSY)
	ECANCELED       Errno = (Errno)(syscall.ECANCELED)
	ECHARSET        Errno = -4080
```